File: test_kamera.cpp
#include <webots/Robot.hpp>
// Added a new include file
#include <webots/Motor.hpp>
#include <webots/Camera.hpp>

#define TIME_STEP 32
#define camera_time_step 32

// All the webots classes are defined in the "webots" namespace

using namespace webots;

int main(int argc, char **argv) {
 Robot *robot = new Robot();

 // get the motor devices
 Motor *leftMotor = robot->getMotor("left wheel motor");
 Motor *rightMotor = robot->getMotor("right wheel motor");
 // set the target position of the motors
 leftMotor->setPosition(10.0);
 rightMotor->setPosition(8.0);
 
 Camera *frontCamera = robot->getCamera("camera");
 frontCamera->enable(32);

 while (robot->step(TIME_STEP) != -1);

 delete robot;

 return 0;
}

File: test_kamera.py
"""kontroleeeeer controller."""

# You may need to import some classes of the controller module. Ex:
#  from controller import Robot, Motor, DistanceSensor
from controller import Robot


# create the Robot instance.
robot = Robot()

# get the time step of the current world.
timestep = int(robot.getBasicTimeStep())

# You should insert a getDevice-like function in order to get the
# instance of a device of the robot. Something like:
#  motor = robot.getMotor('motorname')
#  ds = robot.getDistanceSensor('dsname')
#  ds.enable(timestep)

leftMotor = robot.getMotor('left wheel motor')
rightMotor = robot.getMotor('right wheel motor')

leftMotor.setPosition(float('+inf'))
rightMotor.setPosition(float('+inf'))
leftMotor.setVelocity(0.0)
rightMotor.setVelocity(0.0)
maxSpeed = min(rightMotor.getMaxVelocity(), leftMotor.getMaxVelocity())

leftMotor.setVelocity(5.24)
rightMotor.setVelocity(5.24)

robot.step(300)

leftMotor.setVelocity(5.24)
rightMotor.setVelocity(5.24)

cam = robot.getCamera('camera')
cam.enable(32)







# Main loop:
# - perform simulation steps until Webots is stopping the controller
while robot.step(timestep) != -1:

    # Read the sensors:
    # Enter here functions to read sensor data, like:
    #  val = ds.getValue()

    # Process sensor data here.

    # Enter here functions to send actuator commands, like:
    #  motor.setPosition(10.0)
    pass

# Enter here exit cleanup code.
